libretroshare/src/rs_upnp/UPnPBase.cpp

```diff
@@ -1372,10 +1335,11 @@ int CUPnPControlPoint::Callback(
 	case UPNP_DISCOVERY_SEARCH_TIMEOUT:
 	{
 #if (UPNP_VERSION > 10624) && (UPNP_VERSION < 10800)
-		std::cerr <<  "********************************************************************************" << std::endl
-		          <<  "***                THIS SHOULD NOT HAPPEN !!!  TELL IT TO DEVS               ***" << std::endl
-		          <<  "*** UPnPBase.cpp CUPnPControlPoint::Callback() UPNP_DISCOVERY_SEARCH_TIMEOUT ***" << std::endl
-		          <<  "********************************************************************************" << std::endl;
+		RsErr() << __PRETTY_FUNCTION__
+		        << " ******** (UPNP_VERSION > 10624) && (UPNP_VERSION < 10800) "
+		           " UPNP_DISCOVERY_SEARCH_TIMEOUT  ***************************"
+		           " THIS SHOULD NEVER HAPPEN !!!  TELL IT TO DEVS ************"

```

Comment:
@PhenomRetroShare  This needs clarification https://github.com/RetroShare/RetroShare/issues/1996